sympy/utilities/runtests.py

```diff
@@ -1059,7 +1067,8 @@ def __init__(self, reporter, kw="", post_mortem=False,
         else:
             self._slow_threshold = 10
 
-    def test(self, sort=False, timeout=False, slow=False, enhance_asserts=False):
+    def test(self, sort=False, timeout=False, fail_on_timeout=False, slow=False,
+            enhance_asserts=False):
```

Comment:
To preserve arguments passed positionally, this new parameter should be added to the end of the parameter list, shouldn't it?